robottelo/ui/contentviews.py

```diff
@@ -450,6 +462,41 @@ def add_remove_errata_to_filter(self, cv_name, filter_name,
         else:
             self.click(locators['contentviews.remove_errata'])
 
+    def edit_erratum_date_range_filter(
+            self, cv_name, filter_name, errata_types=None, date_type=None,
+            start_date=None, end_date=None, open_filter=True):
+        """Edit Erratum Date Range Filter"""
+        allowed_errata_types = ['security', 'enhancement', 'bugfix']
+        allowed_date_types = ['updated', 'issued']
+        if not errata_types:
+            errata_types = ['security']
+        if not date_type:
+            date_type = 'updated'
+        if date_type not in allowed_date_types:
+            raise UIError('date type "{0}" does not exit'.format(date_type))
+        if open_filter:
+            self.go_to_filter_page(cv_name, filter_name)
+        for errata_type in allowed_errata_types:
+            errata_type_elm = self.wait_until_element(
+                locators['contentviews.erratum_type_checkbox'] % errata_type)
+            if errata_type_elm:
+                if errata_type_elm.get_attribute('value') == 'true':
```

Comment:
Can we use `self.wait_until_element(locator).is_selected()` instead here? in case that is a checkbox of course. Then probably we will not need that if condition